### PR TITLE
Improve admin overview page in case of error

### PR DESCRIPTION
**Pull request by katyafervent**
Previosly, infinitive loading were shown, but
actual error was not shown to the user

### CODE REVIEW

File: kqueen_ui/blueprints/manager/templates/manager/overview.html [hunk 1]
```diff
@@ -122,29 +122,45 @@ <h3>
 
 {% block extrajs %}
 <script>
+var handleError = function (elId, errorDetails) {
```
Comment (ekhomyakova) [inline kqueen_ui/blueprints/manager/templates/manager/overview.html]: Why don't you show flash message instead?

File: kqueen_ui/blueprints/manager/templates/manager/overview.html [hunk 2]
```diff
@@ -122,29 +122,45 @@ <h3>
...
+  var error =
```
Comment (ekhomyakova) [inline kqueen_ui/blueprints/manager/templates/manager/overview.html]:
```
var error = `
  <tr class='error'>
    <td colspan='6' class='text-center'>
      <i class='mdi mdi-alert-circle-outline'></i>Error occured wile loading the content
    </td>
  </tr>`;
```

File: kqueen_ui/blueprints/manager/templates/manager/overview.html [hunk 3]
```diff
@@ -122,29 +122,45 @@ <h3>
...
+          "<tr class='error' >" +
+          "  <td colspan='6' class='text-center'>" +
+          "    <i class='mdi mdi-alert-circle-outline' s></i> Error occured wile loading the content" +
```
Comment (ekhomyakova) [inline kqueen_ui/blueprints/manager/templates/manager/overview.html]: What is `s` inside of `i` tag?

File: kqueen_ui/blueprints/manager/templates/manager/overview.html [hunk 4]
```diff
@@ -122,29 +122,45 @@ <h3>
...
+          "  </td>" +
+          "</tr>";
+  $(elId + " .not-event-pointer").replaceWith(error);
```
Comment (ekhomyakova) [inline kqueen_ui/blueprints/manager/templates/manager/overview.html]: Please use single quotes for JS

File: kqueen_ui/blueprints/manager/templates/manager/overview.html [hunk 5]
```diff
@@ -122,29 +122,45 @@ <h3>
...
       if (data.response === 200) {
         $('#tableClusters').replaceWith(data.body);
-      }
-    }
-  ).then(function() {
+    } else {
+        console.log(data);
+        handleError('#tableClusters', data.body)
+      }}
+  ).always(function() {
```
Comment (ekhomyakova) [inline kqueen_ui/blueprints/manager/templates/manager/overview.html]: get can fail if some internal error happens (some exception is not catched), so I suggest to add `.fail()` chain as well

**Comment by katyafervent** [comment]
screenshot from 2018-06-20 15-13-59

File: kqueen_ui/blueprints/manager/templates/manager/overview.html (hunk 2 above)
Comment (katyafervent) [reply kqueen_ui/blueprints/manager/templates/manager/overview.html]: :+1:

File: kqueen_ui/blueprints/manager/templates/manager/overview.html (hunk 1 above)
Comment (katyafervent) [reply kqueen_ui/blueprints/manager/templates/manager/overview.html]: it is shown as part of kqueen request exception handling. I'm saving error info to show it to the user on a hover to a failed table

File: kqueen_ui/blueprints/manager/templates/manager/overview.html [hunk 6]
```diff
@@ -122,29 +122,45 @@ <h3>
 
 {% block extrajs %}
 <script>
+var handleError = function (elId, errorDetails) {
+  var error = `
+        <tr>
```
Comment (ekhomyakova) [inline kqueen_ui/blueprints/manager/templates/manager/overview.html]: maybe 2 spaces of indentation?..

**Pull request merged**